robottelo/host_helpers/cli_factory.py

```diff
@@ -1157,3 +1158,25 @@ def setup_cdn_and_custom_repos_content(
             data['lce'] = lce
 
         return data
+
+    def make_medium(self, options=None):
+        """Creates a Medium
+
```

Comment:
it seems dict for make_medium is already defined in ENTITY_FIELDS, so might not need this func

Reply:
Right, updated.